src/main/scala/cognite/spark/v1/AlphaDataModelInstancesRelation.scala

```diff
@@ -26,8 +26,8 @@ import java.time._
 
 import com.cognite.sdk.scala.v1.DataModelType.NodeType
 
-// scalastyle:off cyclomatic.complexity
-class DataModelInstanceRelation(
+// scalastyle:off cyclomatic.complexity file.length
```

Comment:
This means the file starts to get bigger and hard to read, may be it can be factorized by creating one NodeHelper.scala and one EdgeHelper.scala, see if you can move some method inside each object ...